keras/utils/lsuv_init.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import print_function
+import numpy as np
+from keras.models import Model
+from keras.layers import Dense, Convolution2D
+
+
+def svd_orthonormal(shape):
+    # Orthonorm init code is taked from Lasagne
+    # https://github.com/Lasagne/Lasagne/blob/master/lasagne/init.py
```

Comment:
For copyright reasons, don't do this. If you need to port code, please rewrite it from scratch (it's short anyway).